### ISSUE TITLE
Make `map-properties` popup size dynamic

### ISSUE THREAD

**Issue by AliyanH** [Maps4HTML/MapML.js#893]
We should use css to size our map-properties popup, currently if we open a popup on a mobile (or a small map) it takes up the whole screen:
![image](https://github.com/Maps4HTML/Web-Map-Custom-Element/assets/55751566/76551cbd-2511-4954-949a-86ea7c27a4f5)

possibly related to #354

**Comment by AliyanH** [comment]
Example: post the following json into the [sandbox](https://maps4html.org/web-map-doc/demo/sandbox/):

`{"type":"FeatureCollection","crs":{"type":"name","properties":{"name":"EPSG:4326"}},"features":[{"type":"Feature","id":11,"geometry":{"type":"Point","coordinates":[-80.5014667657488,43.4379750604839]},"properties":{"OBJECTID":11,"LANDMARKID":243,"LANDMARK":"ST MARYS GENERAL HOSPITAL","CIVIC_NO":911,"STREET":"QUEENS BLVD","OWNERSHIP":"MINISTRY OF HEALTH","CREATE_DATE":null,"UPDATE_DATE":"Thu, 30 May 2024 15:44:16 GMT","SOURCE":"REGION OF WATERLOO PUBLIC HEALTH WEBSITE","SOURCE_DATE":null,"CATEGORY":"HOSPITAL","SUBCATEGORY":"KITCHENER","STATUS":"ACTIVE","STATUS_DATE":null,"MUNICIPAL_FACILITY":"N","COMMON_LOCATION_REFERENCE":null,"LOCATION_DESCRIPTION":null,"ON_CALL":"N","DIVISION_RESPONSIBLE":"NON-CITY","ASSOCIATED_LAND_AREA":16255.25,"CONSTRUCTION_YEAR":null,"MAP_LABEL":"St Marys"}},{"type":"Feature","id":12,"geometry":{"type":"Point","coordinates":[-80.5122814867122,43.4563721832762]},"properties":{"OBJECTID":12,"LANDMARKID":436,"LANDMARK":"GRAND RIVER HOSPITAL","CIVIC_NO":835,"STREET":"KING ST W","OWNERSHIP":"MINISTRY OF HEALTH","CREATE_DATE":null,"UPDATE_DATE":"Thu, 30 May 2024 15:35:16 GMT","SOURCE":"REGION OF WATERLOO PUBLIC HEALTH WEBSITE","SOURCE_DATE":null,"CATEGORY":"HOSPITAL","SUBCATEGORY":"KITCHENER","STATUS":"ACTIVE","STATUS_DATE":null,"MUNICIPAL_FACILITY":"N","COMMON_LOCATION_REFERENCE":null,"LOCATION_DESCRIPTION":null,"ON_CALL":"N","DIVISION_RESPONSIBLE":"NON-CITY","ASSOCIATED_LAND_AREA":53286.28,"CONSTRUCTION_YEAR":null,"MAP_LABEL":"Grand River"}},{"type":"Feature","id":13,"geometry":{"type":"Point","coordinates":[-80.4122261160498,43.4250518749327]},"properties":{"OBJECTID":13,"LANDMARKID":242,"LANDMARK":"GRAND RIVER HOSPITAL - FREEPORT HEALTH CENTRE","CIVIC_NO":3570,"STREET":"KING ST E","OWNERSHIP":"MINISTRY OF HEALTH","CREATE_DATE":null,"UPDATE_DATE":"Thu, 15 Nov 2018 11:53:45 GMT","SOURCE":"REGION OF WATERLOO PUBLIC HEALTH WEBSITE","SOURCE_DATE":null,"CATEGORY":"HOSPITAL","SUBCATEGORY":"KITCHENER","STATUS":"ACTIVE","STATUS_DATE":null,"MUNICIPAL_FACILITY":"N","COMMON_LOCATION_REFERENCE":null,"LOCATION_DESCRIPTION":null,"ON_CALL":"N","DIVISION_RESPONSIBLE":"NON-CITY","ASSOCIATED_LAND_AREA":78406.96,"CONSTRUCTION_YEAR":null,"MAP_LABEL":"Grand River Hospital - Freeport Health Centre"}}]}`



The map-properties take up the whole screen, instead there should be a scrollable bar, something like _overflow:auto_ https://developer.mozilla.org/en-US/docs/Web/CSS/overflow:
![image](https://github.com/user-attachments/assets/9f2c49a6-a5f5-4a96-8c05-86148d924d8a)
